.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: rsa-keygen Issue
+about: 'Template for submitting an issue'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Pre-requisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+# Expected Behavior
+
+(Please describe the behavior you are expecting)
+
+# Current Behavior
+
+(What is the current behavior?)
+
+# Bug Details
+
+(Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.)
```

Comment:
Instead of just creating single template, there can be multiple templates: https://github.blog/2018-01-25-multiple-issue-and-pull-request-templates/ i.e. one for bug reports, one for feature requests, etc.

Reply:
alright, I'll split it up